topics/metagenomics/tutorials/general-tutorial/tours/metagenomics-general-tutorial-shotgun.yml

```diff
@@ -0,0 +1,326 @@
+      metagenomic sample of the Pampas Soil (<a
+      href="https://www.ebi.ac.uk/metagenomics/projects/SRP016633/samples/SRS372043/runs/SRR606451/results/versions/2.0">SRR606451</a>).
+    backdrop: true
+  - title: Create a new history
+    element: '#history-options-button'
+    content: >-
+      We will start the analyses by creating a new history. Click on this button
+      and then "Create New"
+    placement: left
+  - title: Data upload
+    element: '#tool-panel-upload-button .fa.fa-upload'
+    content: 'We need now to upload the input data. So, open the upload box'
+    placement: right
+    postclick:
+      - '#tool-panel-upload-button .fa.fa-upload'
+      - '#btn-reset'
+  - title: Data upload
+    element: '#btn-new'
+    content: >-
+      Our input datasets are available online on <a
+      href="https://zenodo.org/record/815875">Zenodo</a>. To import them, click
+      on "Paste/Fetch data"
+    placement: right
+    postclick:
+      - '#btn-new'
+  - title: Data upload
+    element: .upload-text-column .upload-text .upload-text-content.form-control
+    content: >-
+      Insert the links to the input file: SRR606451_pampa.fasta.<br><br> You can
+      extract this information from <a
+      href="https://zenodo.org/record/815875">Zenodo</a> with right click on the
+      interesting dataset and then "Copy Link Address"
+    placement: right
+    textinsert: |
+      https://zenodo.org/record/815875/files/SRR606451_pampa.fasta
+  - title: Data upload
+    element: '#btn-start'
+    content: Click on "Start" to upload the data into your Galaxy history.
+    placement: right
+    postclick:
+      - '#btn-start'
+  - title: Data upload
+    element: '#btn-close'
+    content: >-
+      The upload may take a while.<br> Hit the close button to close this
+      window.
+    placement: right
+    postclick:
+      - '#btn-close'
+  - title: Rename the input data
+    element: '.history-right-panel .list-items > *:first'
+    content: >-
+      The uploaded datasets are in the history, but their names correspond to
+      the link. We want to rename them<br><br> For each dataset: <ol>
+        <li>Click on the pencil icon beside the file to "Edit Attributes"</li>
+        <li>Change the name in "Name" to get only the name of the sample</li>
+      </ol> 
+    position: left
+  - title: Extraction of taxonomic information
+    content: >-
+      As for amplicon data, we can extract taxonomic and community structure
+      information from shotgun data. Different approaches can be used: <ul>
+        <li>Same approaches as for amplicon data with identification and classification of OTUs. Such approaches imply a first step of sequence sorting to extract only the 16S and 18S sequences on which the same tools as for amplicon data. However, rRNA sequences represent a low proportion (< 1%) of the shotgun sequences so such approache is not the most statistically supported</li>
```

Comment:
`s/such approache is/such an approach is/` or `s/such approache is/such approaches are/`